addons/binding/org.openhab.binding.network.test/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You are right, I made one test fail on purpose, executed `mvn test` and it still says success. I will now test with an esh test.